lib/Backends/CPU/tests/CPUOperatorTest.cpp

```diff
@@ -0,0 +1,146 @@
+#include "tests/unittests/OperatorTest.h"
+
+#include <iostream>
+
+using namespace glow;
+
+IS_TEST_ENABLED(std::string name) {
+  std::set<std::string> blacklist = {
+      "less_float16Cases/0",
+      "less_int64Cases/0",
```

Comment:
Why do we need to keep this list as code? May be it should be in some kind of a config file?

Formulating this as code has some advantages, because this way it is closer to the actual code that uses it. But I'm not quite convinced if it is really so important. On the negative side, you have to touch the code and rebuild tests if you want to change something.  Using a config file would also allow for using the same file for defining whitelists/blacklists for multiple test suites at once, which could be convenient.

Reply:
I don't think there are enough advantages to a config file to make it worthwhile.

Reply:
Actually, I might relent on the config file front.  I'm starting to worry a bit about the amount of "ceremony" involved in creating all of these backend-specialized tests, and I think it might be good to drive things from a config instead of a bunch of code.  I think I might be able to address your point that it's easy for backends to "overlook" a test suite, too, using a bit of build system sorcery.

I'll try it out tonight and see if it pans out...

Reply:
> I think I might be able to address your point that it's easy for backends to "overlook" a test suite, too, using a bit of build system sorcery.

Hopefully this will work with both CMake and buck.

Reply:
Yeah, CMake is the big concern for me, I'm pretty confident with Buck (although there are some definite gotchas there too).